### PR TITLE
Add KOMPETE to Base Chain

### PR DESCRIPTION
**Pull request by cgallic**
Add KOMPETE token to Base. We are live on mainnet, and have created the token on Base using the Mintable Factory. 

@roberto-bayardo

### CODE REVIEW

File: data/KOMPETE/data.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "symbol": "KOMPETE",
+    "decimals": 10,
+    "description": "KOMPETE's virtual currency that enables players to trade cosmetics with each other on the Bazaar.",
+    "website": "https://kompete.game",
+    "twitter": "@KOMPETEgame",
+    "tokens": {
+      "ethereum": {
+        "address": "0x1E0b2992079B620AA13A7c2E7c88D2e1E18E46E9"
+      },
+
```
Comment (cfluke-cb) [inline data/KOMPETE/data.json:12]: Can you remove the extra empty lines?

**Review by cfluke-cb — DISMISSED** [review]

**Review by cgallic — COMMENTED** [review]
removed

**Review by cfluke-cb — APPROVED** [review]
looks good

**Review by hamdiallam — APPROVED** [review]

**Comment by hamdiallam** [comment]
Hi @cgallic, it looks like the validation script is failing because the erc20 properties differ. Can you address these?

**Comment by cgallic** [comment]
@hamdiallam just updated to make everything matching. On the Ethereum Side our token does only have 10 decimals in the contract. 

When I deployed the base token through mintable ERC 20 I originally forgot the Token part in the name. I've redeployed to fix that. I'm not sure about the decimals though since they deploy with 18.

**Comment by cfluke-cb** [comment]
Hi @cgallic, We will want to have the contract have matching decimals. You will not be able to use the factory, but will need to use the OptimismMintableERC20 contract directly. https://basescan.org/address/0xf23d369d7471bd9f6487e198723eea023389f1d4#code#F2#L1

**Comment by cgallic** [comment]
That is how I created the token for the contract for the base KOMPETE token: 

https://basescan.org/tx/0xb8f0b29a5333b17f93c8ece5f94483ead8352cefda1cff0480f10b7b2ee36efd

On Ethereum we have 9 decimals. 
On Base the contract seems to be auto set to 18 decimals

Is there someway to deploy with 9?

**Review by cfluke-cb — APPROVED** [review]

**Review by roninjin10 — APPROVED** [review]

**Comment by cgallic** [comment]
thank you guys!

**Pull request merged**